gridcapa-task-manager-app/src/main/java/com/farao_community/farao/gridcapa/task_manager/app/ProcessFileRepository.java

```diff
@@ -22,10 +22,10 @@
 @Repository
 public interface ProcessFileRepository extends JpaRepository<ProcessFile, UUID> {
 
-    @Query("SELECT process_file FROM ProcessFile process_file WHERE process_file.fileObjectKey = :fileObjectKey")
+    @Query("SELECT process_file FROM ProcessFile process_file LEFT JOIN FETCH process_file.tasks  WHERE process_file.fileObjectKey = :fileObjectKey")
     Optional<ProcessFile> findByFileObjectKey(@Param("fileObjectKey") String fileObjectKey);
 
-    @Query("SELECT process_file FROM ProcessFile process_file " +
+    @Query("SELECT process_file FROM ProcessFile process_file LEFT JOIN FETCH process_file.tasks " +
```

Comment:
This will create performance issues especially for annual files that create 8760 tasks when arriving

Reply:
true